src/app/app.routes.ts

```diff
@@ -40,6 +41,8 @@ export interface RouteWithData extends Route {
 
 export type RoutesWithData = RouteWithData[];
 
+const fusionauth_host = SecretsService.getStatic('FUSIONAUTH_HOST');
```

Comment:
Little nitpick here: Can we camelCase this variable name?

Reply:
done!